src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/__tests__/HomeStatisticsCardViewModelTests.swift

```diff
@@ -478,7 +698,16 @@ class HomeStatisticsCardViewModelTests: CWATestCase {
 
 				switch rank {
 				case .primary:
-					XCTAssertEqual(viewModel.primaryValue, expectedString)
+					switch HomeStatisticsCard(rawValue: id) {
+					case .atLeastOneVaccinatedPerson, .fullyVaccinatedPeople:
+						if inPercent {
+							XCTAssertEqual(viewModel.primaryValue, expectedString + AppStrings.Statistics.percent)
```

Comment:
```suggestion
							XCTAssertEqual(viewModel.primaryValue, expectedString + "%")
```

For the tests you can use the percent symbol directly and I think then you can throw away the app string and the localizable string.